prdoc/pr_4902.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Removed `pallet::getter` usage from the pallet-vesting
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removed `pallet::getter`s from `pallet-vesting`s storage items.
+      When accessed inside the pallet, use the syntax `StorageItem::<T>::get()`.
```

Comment:
It's not only when accessed inside the pallet

Reply:
I haven’t mentioned the process for outside the pallet because it remains the same by calling the public function. 
Did I answer your question correctly, or did I misunderstand it?